### ISSUE TITLE
tqdm

### ISSUE THREAD

**Issue by cmacdonald** [Georgetown-IR-Lab/cedr#38]
Sean, there is some peculiar tqdm setups in CEDR. Could we remove the leave=false and the ncols; perhaps set the units and some descriptions? 

If you agree, I can submit a PR.

**Comment by seanmacavaney** [comment]
Yes, adding units and writing better descriptions would be helpful.

The `leave` and `ncols` settings are important for the usability of the tool on the command line. Removing ncols makes the tqdms really ugly (and the tool borderline unusable if you dare resize your console), and setting `leave=False` makes it much easier to scan the train/validation performance over time without dead tqdms getting in the way (which themselves no longer provide much/any value).

I know that ncols in notebooks produces some pretty ugly bars (it's the number of pixels, not the number of characters). What effect does `leave=False` have? Is there an easy way to detect that it's running in a notebook? If so, these could be set conditionally.

**Comment by cmacdonald** [comment]
leave=False means the progress bar dissapears after use. If tqdm is obtained in imported in one single place, it could be overwritten?

**Comment by seanmacavaney** [comment]
That sounds fair. So tqdm could be set with CLI defaults in CEDR, and then PyTerrier could overwrite that reference.
